cfgov/v1/forms.py

```diff
@@ -161,6 +161,7 @@ def render_with_id(self, field, attr_id):
         for f in self.fields:
             if field.html_name == f:
                 self.fields[f].widget.attrs.update({'id': attr_id})
```

Comment:
you can combine this with 163 like `.update({'id': attr_id, 'class': 'a-text-input'}) `